### PR TITLE
Redesign the Demo Page

### PR DESCRIPTION
**Pull request by Harsh-Betty**

### CODE REVIEW

File: components/Cards.vue [hunk 1]
```diff
@@ -120,4 +112,34 @@ const popup = [
     //         'A short description about the Hide XP and cFuel Widget',
     // },
 ]
+
+const people = [
```
Comment (rmagon) [inline components/Cards.vue]: Where do we need people?

File: pages/sidebar/simple.vue [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
-          </div>
-        </div>
-        <div
-          class="absolute inset-x-0 top-[calc(100%-13rem)] -z-10 transform-gpu overflow-hidden blur-3xl sm:top-[calc(100%-30rem)]"
-          aria-hidden="true">
-          <div
-            class="relative left-[calc(50%+3rem)] aspect-[1155/678] w-[36.125rem] -translate-x-1/2 bg-gradient-to-tr from-[#ff80b5] to-[#9089fc] opacity-30 sm:left-[calc(50%+36rem)] sm:w-[72.1875rem]"
-            style="clip-path: polygon(74.1% 44.1%, 100% 61.6%, 97.5% 26.9%, 85.5% 0.1%, 80.7% 2%, 72.5% 32.5%, 60.2% 62.4%, 52.4% 68.1%, 47.5% 58.3%, 45.2% 34.5%, 27.5% 76.7%, 0.1% 64.9%, 17.9% 100%, 27.6% 76.8%, 76.1% 97.7%, 74.1% 44.1%)" />
-        </div>
+  <div class="bg-purple-custom">
```
Comment (rmagon) [inline pages/sidebar/simple.vue:2]: Let's move all of it in a component called Hero. And take the title and subtitle through props https://vuejs.org/guide/components/props.html

File: pages/sidebar/simple.vue [hunk 3]
```diff
@@ -1,32 +1,33 @@
...
+    <div class="max-w-7xl mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:px-8">
+      <div class="text-center mt-10">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor"
```
Comment (rmagon) [inline pages/sidebar/simple.vue]: Please put the SVGs in separate files and import them where you want to use them

File: components/Cards.vue (hunk 1 above)
Comment (Harsh-Betty) [reply components/Cards.vue]: I was using it as a placeholder, forgot to remove it before pushing.

**Pull request merged**